packages/qwik/src/core/render/jsx/types/jsx-generated.ts

```diff
@@ -599,37 +599,9 @@ type SpecialAttrs = {
     value?: string | ReadonlyArray<string> | number | undefined | null | FormDataEntryValue;
     width?: Size | undefined;
     children?: undefined;
-  } & (
-    | {
-        type?:
-          | Exclude<HTMLInputTypeAttribute, 'button' | 'reset' | 'submit' | 'checkbox' | 'radio'>
-          | undefined;
-        'bind:checked'?: undefined;
-      }
-    | {
-        type: 'button' | 'reset' | 'submit';
-        'bind:value'?: undefined;
-        'bind:checked'?: undefined;
-        autoComplete?: undefined;
-      }
-    | {
-        type: 'checkbox' | 'radio';
-        'bind:value'?: undefined;
-        autoComplete?: undefined;
-      }
-  ) &
-    (
-      | {
-          type?: Exclude<HTMLInputTypeAttribute, 'button'> | undefined;
-          popovertarget?: undefined;
-          popovertargetaction?: undefined;
-        }
-      | {
-          type: 'button';
-          popovertarget?: string | undefined;
-          popovertargetaction?: PopoverTargetAction | undefined;
-        }
-    );
+    popovertarget?: string | undefined;
+    popovertargetaction?: PopoverTargetAction | undefined;
```

Comment:
you left out the bind and autocomplete ones. Please add /** */ comments to them to explain they're only for those types

Reply:
Not sure if this is what you wanted, but I added some comments. 

Btw, input with type checkbox/radio can actually use the value attribute: https://www.w3schools.com/tags/att_input_value.asp